src/uu/mv/src/mv.rs

```diff
@@ -251,6 +251,17 @@ fn parse_paths(files: &[OsString], b: &Behavior) -> Vec<PathBuf> {
 }
 
 fn handle_two_paths(source: &Path, target: &Path, b: &Behavior) -> UResult<()> {
+    if b.backup == BackupMode::SimpleBackup && source_is_dst_backup(source, target, &b.suffix) {
+        return Err(io::Error::new(
+            io::ErrorKind::NotFound,
+            format!(
+                "backing up {} might destroy source;  {} not moved",
```

Comment:
Here, the `b` is lowercase, but in the test it's uppercase, but somehow the test still passes? How does that work?

Reply:
because of 
https://github.com/uutils/coreutils/blob/ab8aa32e60602aef882daa561263d82086faee71/src/uucore/src/lib/mods/error.rs#L426-L427
maybe we should remove this

Reply:
I had no idea we were doing capitalization. Removing it sounds like a good idea. Making things less magical is always good :)

Reply:
fixed in https://github.com/uutils/coreutils/pull/5004